env.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-awk -F '=' '{ print $1 ": \"" (ENVIRON[$1] ? ENVIRON[$1] : $2) "\"" }' ./application.docker.yml >> ./config/application.yml
+awk -F '=' '{ print $1 ": \"" (ENVIRON["ADMIN_UI"] ? ENVIRON["ADMIN_UI"] : $2) "\"" }' ./application.docker.yml >> ./config/application.yml
```

Comment:
remove ./application.docker.yml and just add the end of ./config/application.yml ENVIRON["ADMIN_UI"]